etc/config/fpc.defaults.properties

```diff
@@ -0,0 +1,19 @@
+# Default settings for FPC
+compilers=/usr/bin/fpc
+defaultCompiler=/usr/bin/fpc
+compileFilename=output.pas
+postProcess=
+demangler=
+objdumper=objdump
+options=
+versionFlag=-h | grep "Compiler version" 
+supportsBinary=true
```

Comment:
I think you set supportsBinary to false every time, right? (fpc.js L#6) Why is then this set to true as defaul?

Reply:
I just checked it with this change, but settings this to false causes the disassembly window to show binary ELF... etc instead of assembly, and setting the setting in fpc.js doesn't really do anything, so I removed it there. It does need to be true here.

Reply:
Sorry for not closing this until now. Did not realize this was still open. If this need sot be true here, then so be it! :)